### PR TITLE
feat(analysisfile): create model and migration

### PR DESCRIPTION
**Pull request by adrian-lara**
[Waiting for the AnalysisRun model to be merged to develop]

#### Any background context you want to provide?
Part of the larger Analysis epic

#### What's this PR do?
Creates the AnalysisFile model and corresponding migration

#### How should this be manually tested?
Run migration

#### What are the relevant tickets?
#2439

### CODE REVIEW

File: seed/models/analysis_files.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    BUILDINGSYNC = 1
+
+    FILE_TYPES = (
+        (BUILDINGSYNC, 'BuildingSync'),
+    )
+
+    file = models.FileField(upload_to="analysis_files", max_length=500)
+    file_type = models.IntegerField(choices=FILE_TYPES)
+    input_for_run = models.ForeignKey(AnalysisRun, on_delete=models.CASCADE, related_name='input_files', blank=True, null=True)
+    output_for_run = models.ForeignKey(AnalysisRun, on_delete=models.CASCADE, related_name='output_files', blank=True, null=True)
```
Comment (adrian-lara) [inline seed/models/analysis_files.py:18]: Looking for feedback and or other ideas on the `_for_run` names. Technically, an individual AnalysisFile is an "input/output for a run".

**Comment by adrian-lara** [comment]
I'm wondering if anyone has an opinion on whether we should have a follow up PR to this that combines the 3 Analysis migrations together into one migration. Thoughts??

**Comment by macintoshpie** [comment]
> I'm wondering if anyone has an opinion on whether we should have a follow up PR to this that combines the 3 Analysis migrations together into one migration. Thoughts??

Hmm from my point of viewI don't see a great advantage of merging these. It's very database specific and automated, I view it kind of like the commits of the database, and each of these migrations is a meaningful and distinct "commit". The advantages I can think of from merging would be:
- fewer files (marginal advantage since this is in a directory of mostly auto-generated files)
- if rolling back, with desire to undo analysis stuff, only need to care about 1 migration

I could go either way I guess, but might prefer keeping them separate unless theres a really good reason

File: seed/models/analysis_files.py (hunk 1 above)
Comment (macintoshpie) [reply seed/models/analysis_files.py:18]: I can't think of any better ones, I like this 👍

**Review by macintoshpie — APPROVED** [review]
Looks great 👍

**Pull request merged**